lib/graphql/analysis/max_query_depth.rb

```diff
@@ -12,7 +13,7 @@ module Analysis
     #
     class MaxQueryDepth < GraphQL::Analysis::QueryDepth
       def initialize(max_depth)
-        disallow_excessive_depth = ->(query, depth) {
+        disallow_excessive_depth = -> (query, depth) {
```

Comment:
The no-space Procs were originially introduced because someone wanted to support Ruby 1.9. AFAIK they gave up, and don't actually use GraphQL-Ruby after all. So this will be a big change, but probably OK.